rudder-server-relay/SOURCES/relay-api/relay_api/remote_run.py

```diff
@@ -23,7 +23,7 @@
 
 NEXTHOP = None
 REMOTE_RUN_COMMAND = "sudo /opt/rudder/bin/rudder remote run"
-LOCAL_RUN_COMMAND = "sudo /opt/rudder/bin/rudder agent run"
+LOCAL_RUN_COMMAND = "sudo /opt/rudder/bin/rudder agent run &> /dev/null"
```

Comment:
```&>``` is not available in all shell, and we should use ``` > /dev/nul 2>&1 ``` instead ... But maybe &> is available everywhere ? because it seems better to me ...